content/en/privacy-policy/index.md

```diff
@@ -7,4 +7,16 @@ draft: false
 images: []
 ---
 
-# Statistics collected by the update checker
+We do not use cookies, and we do not collect any personal data.
+
+## Statistics collected by the update checker
+
+Currently, no data is being recorded. In the past the following information was recorded when a check for updates was made:
```

Comment:
I guess this is no longer true, as the update server is keeping logs. Maybe just revert to the version on the web archive?